pkgs/misc/vim-plugins/default.nix

```diff
@@ -1015,7 +1015,7 @@ let
       sha256 = "03sr53680kcwxaa5xbqzdfbsgday3bkzja33wym49w9gjmlaa320";
     };
     dependencies = ["vimproc" "vimshell" "self" "forms"];
-    pythonDependencies = with pythonPackages; [ sexpdata websocket_client ];
+    passthru.python3Dependencies = ps: with ps; [ sexpdata websocket_client ];
```

Comment:
This has to be also added to `pkgs/misc/vim-plugins/vim2nix/additional-nix-code`
Is this compatible with vim?